utilities/ifix-migration-toolkit/src/main/java/org/egov/ifixmigrationtoolkit/util/MigrationEnum.java

```diff
@@ -0,0 +1,6 @@
+package org.egov.ifixmigrationtoolkit.util;
+
+public enum MigrationEnum {
```

Comment:
If this is specific only to Department Entity Migration, then use the prefix Departmententity and name it "DepartmentEntityMigrationEnum".

Reply:
minor change, to be taken up as part of code cleanup.